src/components/Viewport.ts

```diff
@@ -483,13 +484,18 @@ export default class Viewport {
     cameraElement.innerHTML = status.panels.map(panel => panel.html).join("");
 
     this.createPanels();
-    this.currentPanel = panelManager.get(status.index);
 
     // Reset panel index
     panelManager.originalPanels().forEach((panel, idx) => {
       panel.setIndex(panels[idx].index);
     });
 
+    this.currentPanel = panelManager.has(status.index)
```

Comment:
@WoodNeck 
How about `panelManager.get(status.index) || panelManager.get(defaultIndex) || panelManger.firstPanel()`?

Reply:
That looks more great, I'll fix it.